crates/ruma-client-api/src/sync/sync_events/v4.rs

```diff
@@ -325,6 +351,18 @@ pub struct SlidingSyncRoom {
     /// The prev_batch allowing you to paginate through the messages before the given ones.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub prev_batch: Option<String>,
+
+    /// True if the number of events returned was limited by the limit on the filter.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub limited: Option<bool>,
```

Comment:
Is there a semantic difference between `None` and `Some(false)`? If not, this should be just a `bool` with `#[serde(default, skip_serializing_if = "ruma_common::serde::is_default")]`.